.github/workflows/lockdown.yml

```diff
@@ -0,0 +1,23 @@
+name: RepoLockdown
+on:
+  issues:
+    types: opened
+  pull_request_target:
+    types: opened
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/repo-lockdown@v2
+       with:
+          issue-comment: >
+            This repository does not accept bug reports,
+            see the README for details.
```

Comment:
I suggest the following text:

New issues are no longer accepted in this repository.  If `singularity --version` says `singularity-ce`, submit instead to https://github.com/sylabs/singularity, otherwise submit to https://github.com/apptainer/apptainer.

Reply:
OK the text has been updated to this new text. Thanks.